src/circuits/atomic-query-v3-on-chain.ts

```diff
@@ -420,7 +422,7 @@ interface AtomicQueryV3OnChainCircuitInputs {
  * @beta
  * AtomicQueryV3OnChainPubSignals public inputs
  */
-export class AtomicQueryV3OnChainPubSignals extends BaseConfig {
+export class AtomicQueryV3OnChainPubSignals extends BaseConfig implements IGistRootStatePubSignals {
```

Comment:
let's rename IGistRootStatePubSignals to IStateInfoPubSignals

Reply:
renamed